src/config.ts

```diff
@@ -17,21 +17,29 @@ export class KubeConfig {
     // Only public for testing.
     public static findHomeDir(): string | null {
         if (process.env.HOME) {
-            if (fs.existsSync(process.env.HOME)) {
+            try {
+                fs.accessSync(process.env.HOME);
```

Comment:
I'm not sure why `existsSync` is being changed to `try { accessSync } catch { /* ignore */ }` as part of this PR.  Assuming there's a good reason, let's not clutter this flow with exception swallowing - let's create a helpfully named wrapper function e.g. `canAccess` or `existsPlusPlus` or whatever your intent is.

Reply:
This was originally done here:

https://github.com/kubernetes-client/javascript/pull/164

To fix the node 10 tests which were breaking.

I've added the cleanup you requested.